src/domain/data-decoder/entities/schemas/data-decoded.schema.ts

```diff
@@ -0,0 +1,31 @@
+import { Schema } from 'ajv';
+
+export const dataDecodedParameterSchema: Schema = {
+  type: 'object',
+  properties: {
+    name: { type: 'string' },
+    param_type: { type: 'string' },
+    value: { anyOf: [{ type: 'string' }, { type: 'number' }] },
+    valueDecoded: {
+      anyOf: [
+        { type: 'object', nullable: true },
+        { type: 'array', nullable: true },
+      ],
+    },
+  },
+  required: ['name', 'value'],
+  additionalProperties: true,
+};
+
+export const dataDecodedSchema: Schema = {
+  type: 'object',
+  properties: {
+    method: { type: 'string' },
+    parameters: {
+      anyOf: [
+        { type: 'null' },
+        { type: 'array', items: { $ref: 'dataDecodedParameter' } },
+      ],
+    },
```

Comment:
Isn't it possible to use `nullable` here instead of having a one of the possible types as `null`?

Reply:
I'm testing payloads in https://www.jsonschemavalidator.net/ and it seems nullable doesn't work in the same way for nested objects.
![Screenshot 2022-10-18 at 12 11 28](https://user-images.githubusercontent.com/21182069/196404015-f3ac5e24-695c-4ede-bf33-2535395bf898.png)
![Screenshot 2022-10-18 at 12 12 33](https://user-images.githubusercontent.com/21182069/196404020-9d60b8f5-5c6e-4f67-8983-d57a050f10f8.png)